app/mutations/application_review_step/deleteReviewCommentMutation.ts

```diff
@@ -12,16 +12,16 @@ import {
 import {ConnectionHandler} from 'relay-runtime';
 
 const mutation = graphql`
-  mutation deleteReviewCommentMutation(
-    $input: UpdateReviewCommentInput!
-    $applicationId: ID!
-    $version: String!
-  ) {
+  mutation deleteReviewCommentMutation($input: UpdateReviewCommentInput!) {
     updateReviewComment(input: $input) {
-      query {
-        application(id: $applicationId) {
-          ...ApplicationDetailsContainer_application
-            @arguments(version: $version)
+      reviewComment {
+        id
+        description
+        createdAt
+        resolved
+        ciipUserByCreatedBy {
+          firstName
+          lastName
```

Comment:
Is all of this needed, or just the `id`?

Reply:
I don't think it is.